_includes/ocrtutorial.md

```diff
@@ -0,0 +1,92 @@
+## OCR Tutorial
+
+This tutorial explains the [Optical Character Recognition (OCR) demo](https://github.com/nimbella/demo-projects/tree/master/ocr) available on GitHub and shows you how to deploy it to the Nimbella Cloud.
+
+The OCR application displays a web page where visitors select a language (English, Spanish, or Chinese) and then drag a JPG, GIF, or PNG image onto the page, where it is automatically converted to text. If the conversion is successful, users can read the text and click a **Speak** button to hear the text. You can [try out the application here](https://ocrdemo-apigcp.nimbella.io).
+
+The OCR demo is an example of a serverless Nimbella Cloud application with a [React front end](https://reactjs.org) and several functions on the back end. The [Tesseract OCR Engine](https://github.com/tesseract-ocr/tesseract) converts the images to text.
+
+The OCR demo has the following code and configuration components, described in more detail later:
+
+-Application logic: The _packages_ directory contains several Nimbella actions, which are logical collections of functions.
+- Static web content: The _web_ directory contains a _public_ directory, with the files that will be part of the public website, and a _src_ directory with React component files and other JavaScript files that are built into the _index.html_ template page to produce the final static public page.
+- Slack application: A Nimbella Commander project folder called _commander_, with an _ocr.js_ file with an app that can be installed into Slack.
```

Comment:
oy - I forgot about these. I think it's better to not document these right now. There is both a workflow and a slack slash command in here and they need more context. Sorry about that.

Reply:
I'm not sure what to do about this, because further down you mention the workflow might stay in, but it requires the Slack app to work. It seems ok to me to leave the Slack app in there if I put in a link to the Nimbella Commander portion of the site, or the Nimbella Commander documentation (I don't have a link for that though). What do you think?